### PR TITLE
#165009755 Update heroku api key (travis) with minimal clean up

### PR DESCRIPTION
**Pull request by devdbrandy**
#### What does this PR do?

This provides a new encrypted api token for heroku with support for `travis-ci.com` API.

#### Description of Task to be completed

- [x] Generated new api token for heroku
- [x] Refactor code base to resolve houndCI warning

#### Any background context you want to provide

n/a

#### What are the relevant pivotal tracker stories

[#165009755](https://www.pivotaltracker.com/story/show/165009755)

#### Screenshots

n/a

### CODE REVIEW

**Review by aanchirinah — APPROVED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ notifications:
...
+    secure: LSBCTDe94MVR8V7usyQBstsbd165GN6Gmh6/CAc5I61oVT1RPBY2hKeacT0rwfCS9y4pIpXrUBQyMZbA7nEUtOulsQJ4i8T0Oin5dqSchJAhWJOdOtgNE2EQrbKVc4Xoz2AfA0uci7/icAf4mEfqFi0MPB8qBEkvxsq9FyceMuINdsYRF+hWbsgyr42xUtuTPLBQnQP+DD+hhkyQQXUZTxOXa7VUQ0A8X8EEZ6oZigKLApMf1cwafzf64vaTxX5IXv03OWDPe0f1ekNdULFPCEcQSc2aeSRsNnVXMtM8v0e+ZZ3NzHswgp8+g15l75v5FtbUXTVo4510Br6DKkRjoLRCqf/1XltlWwfBwkHGMOzE3+9XPvtsu5NhbY9qEese+E/x+khz6mZh5U223YVpHfhaeq3NfoTs9wLfMaq+FCZv3tKFWVDeBH6fHbDS5h/mdequS8GddsuoA8kWQPzOzEo1KOf7QZ6C5cXAFjJRDkUWvduUeCFaUgcxDiRvTR0aJecyi8hrZ24UHdwP49Y+tjCImiBpSDYozEDTEtEQlS/TyYfPGGgj1ntGg1cfsIvhRkDje46WIa3kqkbsm8xfJF8Bk6T8F583GcONOLxh/Q7po6nROsnUvt9BJDDLVLbA+zDqCJa+nthBjS6IpFV87gmbbwjOItoGun5bMvID15s=
```
Comment (valenstical) [inline .travis.yml]: Great work @devdbrandy,
Would it be possible to add the heroku API key as an environment variable on Travis CI? You could then reference that variable on this .travis.yml file ($HEROKU_API_KEY). That way, if we have any reason to regenerate the Heroku API key, we would not need to update this file, commit and push changes to github. Instead, we could just update the key on Travis CI only and everything works smoothly.

Let me know if this is something you can do.

**Review by valenstical — CHANGES_REQUESTED** [review]
Great work @devdbrandy, kindly respond to my requested changes.

File: .travis.yml (hunk 1 above)
Comment (devdbrandy) [reply .travis.yml]: Hi @valenstical, you have made very good point. However, its rather too early in travisCI's sequence for environment variable to be read, which is why travis has provided the ability for us to encrypt sensitive data such as api credentials.

Comment (valenstical) [reply .travis.yml]: Oh, that sounds good. Thanks for clarifying that.

**Review by valenstical — APPROVED** [review]
LGTM

**Comment by devdbrandy** [comment]
Hi @aanchirinah I just realised that the build result is similar to what we have [here](https://github.com/andela/apollo-ah-backend/pull/53#issuecomment-478396808). 

Moving forward I have regenerated a long-term api token (never expires), and to help debug the issue, I will temporarily create a pipeline for the branch

**Comment by devdbrandy** [comment]
Closing PR since heroku has been configured to handle deployment.

**Pull request closed without merging**